zebedee-cms/src/main/java/com/github/onsdigital/zebedee/util/slack/PostMessageField.java

```diff
@@ -0,0 +1,22 @@
+package com.github.onsdigital.zebedee.util.slack;
+
+import com.google.gson.annotations.SerializedName;
+
+public class PostMessageField {
+    public String title;
+    public String value;
+
+    @SerializedName("short")
+    public boolean isShort;
+
+    public PostMessageField(String title, String value) {
+        this.title = title;
+        this.value = value;
```

Comment:
In this case `isShort`will have a default value of `false` - I assume that is OK? Also a common pattern for multiple constructors is to call the the "full" one passing in defaults for the missing values:
```
PostMessageField(String title, String value) {
    self(title, value, false)
}